app/src/main/java/com/indoornavigation/Map.java

```diff
@@ -0,0 +1,39 @@
+package com.indoornavigation;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+
+import android.graphics.Canvas;
+import android.util.AttributeSet;
+
+public class Map extends CanvasView
```

Comment:
To nie zadziala. Dostaniemy wyjatek przy probie infalteowania layoutu. Ta klasa powinna wygladac troche inaczej